core/src/main/scala/com/whisk/docker/DockerKit.scala

```diff
@@ -14,7 +14,7 @@ trait DockerKit {
   private lazy val log = LoggerFactory.getLogger(this.getClass)
 
   val PullImagesTimeout = 20.minutes
-  val StartContainersTimeout = 20.seconds
+  val StartContainersTimeout = 60.seconds
```

Comment:
I wouldn't change defaults here. It should be possible to override it in `DependencyGraphReadyCheckSpec `